balloc/balloc.h

```diff
@@ -161,6 +161,14 @@ struct m0_balloc_group_info {
 	struct m0_lext              *bgi_extents;
 	/** per-group lock */
 	struct m0_be_mutex           bgi_mutex;
+
+	/** balloc allocation context list head */
+	struct m0_list               bgi_bac_head;
```

Comment:
If possible, m0_tlist should be used instead of m0_list.
The same for "struct m0_list_link            bac_link; "

m0_tlist will enforce some type checking.

Reply:
ok